hazelcast/src/main/java/com/hazelcast/aggregation/impl/AbstractAggregator.java

```diff
@@ -97,10 +97,10 @@ public final void accumulate(I entry) {
      * Accumulates a single extracted value.
      * This method may be called multiple times per accumulated entry if the attributePath contains [any] operator.
      *
+     * @param entry The entry containing the value.
      * @param value If attributePath is not null the methods accumulates the value extracted from the attributePath.
      *              If attributePath is null and the input object is a Map.Entry the method accumulates Map.Entry.getValue().
-     *              Otherwise the method accumulates the input value as-is.
      */
-    protected abstract void accumulateExtracted(E value);
+    protected abstract void accumulateExtracted(I entry, E value);
```

Comment:
perhaps better to have two overloads here? the default implementation of `accumulateExtracted(I Entry, E value)` would just call `accumulateExtracted(E value)`. This is an `impl` class but there might still be people overriding it?